internal/process/provisioning/cluster_regions.go

```diff
@@ -0,0 +1,44 @@
+package provisioning
+
+var clusterRegions = []string{
+	"centralus",
+	"eastus",
+	"westus2",
+	"northeurope",
+	"uksouth",
+	"japaneast",
+	"southeastasia",
+	"westeurope",
+	"australiaeast",
+	"switzerlandnorth",
+	"brazilsouth",
+	"canadacentral",
+	"eu-central-1",
+	"eu-west-2",
+	"ca-central-1",
+	"sa-east-1",
+	"us-east-1",
+	"us-west-1",
+	"ap-northeast-1",
+	"ap-northeast-2",
+	"ap-south-1",
+	"ap-southeast-1",
+	"ap-southeast-2",
+	"us-west-2",
+	"eu-central-1",
+	"us-east-1",
+	"ap-southeast-1",
+	"europe-west3",
+	"us-central1",
+	"asia-south1",
+	"asia-northeast2",
+	"me-central2",
+	"me-west1",
+	"australia-southeast1",
+	"southamerica-east1",
+	"asia-northeast1",
+	"asia-southeast1",
+	"us-west1",
+	"us-east4",
+	"europe-west4",
+}
```

Comment:
How come:
```
INPUT: PlanID: 4deee563-e5ec-4731-b9b1-53b42d855f0c, PlatformRegion: cf-ap21, ClusterRegion: centralus, Provider: AWS
!dirty,!euAccess,hyperscalerType=aws,tenantName=e8f7ec0a-0cd6-41f0-905d-5d1efa9fb6c4
INPUT: PlanID: 4deee563-e5ec-4731-b9b1-53b42d855f0c, PlatformRegion: cf-ap21, ClusterRegion: centralus, Provider: Azure
!dirty,!euAccess,hyperscalerType=azure,tenantName=e8f7ec0a-0cd6-41f0-905d-5d1efa9fb6c4
INPUT: PlanID: 4deee563-e5ec-4731-b9b1-53b42d855f0c, PlatformRegion: cf-ap21, ClusterRegion: centralus, Provider: GCP
!dirty,!euAccess,hyperscalerType=gcp,tenantName=e8f7ec0a-0cd6-41f0-905d-5d1efa9fb6c4
```
`centralus` is only valid for `azure` provider (and related plans).

Reply:
This is not issue but could be.
If we have difference for not allowed combinations, we report failure of comparison however it is not necessarily a problem.

Reply:
And here we generate over 100k cases instead of around 3k.

Reply:
It was easier for just to go through all combinations. Test will be removed with removal of old resolve_creds steps.